src/scripts/main.js

```diff
@@ -1,5 +1,17 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+const pushNotification = (topPst, rightPst, title, description, type) => {
```

Comment:
I have not idea what 'Pst' stands for. Perhaps 'position'? Pos is a standard abbreviation for position. 

top, right should be sufficient. Why have you decided to rename them?

Reply:
![image](https://user-images.githubusercontent.com/39385053/100347897-c93ef380-2fee-11eb-9267-a78e8262b2f9.png)
I've decided to rename the abbreviation as the linter said that such value already declared and I can't find where it was.

Reply:
chatted offline. top is global from window.top. keeping unique name.